modules/standard/AdvancedIters.chpl

```diff
@@ -34,12 +42,17 @@ record vlock {
   }
 }
 
+/*
+   Toggle debugging output.
+*/
 config param debugAdvancedIters:bool=false;
 
 //************************* Dynamic iterator
 
-// Serial iterator 
-
+/*
+   The serial version of the dynamic iterator.
+   Equivalent to serial iteration over the range ``c``.
```

Comment:
There needs to be a blank line between these sentences to get a line break in the output.